crates/ruff/src/rules/flake8_bugbear/rules/duplicate_exceptions.rs

```diff
@@ -94,7 +94,7 @@ fn duplicate_handler_exceptions<'a>(
             );
             if checker.patch(diagnostic.kind.rule()) {
                 #[allow(deprecated)]
-                diagnostic.set_fix(Fix::unspecified(Edit::range_replacement(
+                diagnostic.set_fix(Fix::maybe_correct(Edit::range_replacement(
```

Comment:
Is there a specific rule to follow when changing to `safe` or `maybe_correct`, right now I just went off the issue description where it might introduce semantic changes

Reply:
fyi the names have changed per https://github.com/charliermarsh/ruff/issues/4184#issuecomment-1546782684

The meaning for each applicability level is documented in the `Applicability` enum

Reply:
Yeah it makes sense now.